src/toolbars/TimeSignatureToolBar.cpp

```diff
@@ -138,7 +143,7 @@ void TimeSignatureToolBar::Populate()
       this, wxID_ANY,
       wxString::Format(
          "%d", ProjectTimeSignature::Get(mProject).GetLowerTimeSignature()),
-      wxDefaultPosition, size,
+      wxDefaultPosition, timeSigSize,
```

Comment:
Trying to run it on Mac, I got strange looking results for the lts —  but my environment was not clean.  I deleted Audacity.cfg and tried again, and saw 4 not blank, and the down arrow of the combo looked more normal.  But I see rounded corners and slightly different height mismatching the tempo and Uts boxes which are square and positioned a bit higher.

Reply:
Fun fact - wxComboCtr and wxComboBox look disturbingly differently on macOS.

Reply:
I've tried to do wxComboCtrl, but it is not focusable in read-only mode. So I have just adjusted the height to make it look a tiny bit better on macOS